src/main/src/main/java/org/geoserver/GeoserverInitStartupListener.java

```diff
@@ -90,6 +90,11 @@ public void contextInitialized(ServletContextEvent sce) {
         LOGGER.config("Logging policy: " + policy);
         GeoTools.init((Hints) null);
 
+        // enable JTS overlay-ng unless otherwise set
```

Comment:
Note to self, move it up to make sure no other JTS usage can be done before this.